src/vcProject.h

```diff
@@ -37,4 +39,6 @@ bool vcProject_UseProjectionFromItem(vcState *pProgramState, vcSceneItem *pItem)
 bool vcProject_UpdateNodeGeometryFromCartesian(vdkProject *pProject, vdkProjectNode *pNode, const udGeoZone &zone, vdkProjectGeometryType newType, udDouble3 *pPoints, int numPoints);
 bool vcProject_FetchNodeGeometryAsCartesian(vdkProject *pProject, vdkProjectNode *pNode, const udGeoZone &zone, udDouble3 **ppPoints, int *pNumPoints);
 
+const char *vcProject_GetFixedPath(vdkProject *pProject, const char *pPath);
```

Comment:
This isn't defined in this PR?

Reply:
Yeah- I removed it form this PR but missed the header apparently